### DISCUSSION TITLE
is openintegrationhub moving away from opensource?

### DISCUSSION THREAD

**Discussion by voodark** [openintegrationhub/openintegrationhub#1522] [Q&A] [answered]
I saw on net " Open Integration Hub is becoming FlowMate!". is FlowMate aquiring openintegrationhub ?

**Comment by RobinBrinkmann** [comment, marked as answer]
Hey @voodark, thanks for the question! 

The open source framework will stay open and under Apache 2.0, allowing for broad use in commercial products. FlowMate is one commercial implementation of the framework and offering a platform for SaaS vendors to offer integration from within their product. FlowMate however has it's offspring in what was called "Open Integration Hub Platform" - which was an effort to create a productised version of the open source framework.

**Reply by voodark** [reply]
Thank you
